frontend/i18next.scanner/pt/translation.json

```diff
@@ -1,5 +1,6 @@
 {
   "{{author}} added you to {{space}}": "{{author}} adicionou-o à {{space}}",
+  "{{author}} joined space {{space}}": "{{author}} juntou-se ao espaço {{space}}",
```

Comment:
This Portuguese translation doesn't match the specified one (which is "X juntou o espaço Y" in the issue)

But I'm not qualified to judge which is the best

Reply:
Issue updated.